deploy-nagios/tasks/main.yml

```diff
@@ -18,23 +18,39 @@
 -
   name: "Get jenkins url"
   shell: "oc get route jenkins --namespace={{ project_name }} -o jsonpath='{.spec.host}'"
+  changed_when: False
   register: route_output
 
 -
   name: "Set jenkins url"
   set_fact:
-    jenkins_url: "{{ route_output.stdout }}"
-  when: route_output | changed
+    jenkins_url: "{{ route_output.stdout | default('') }}"
 
 -
   name: "Get image version"
   set_fact:
     nagios_image_version: "{{ nagios_image_def[0][0].image }}"
 
 -
+  name: "Set value for platforms"
+  set_fact:
+    digger_platforms: "{{ digger_platforms if digger_platforms is defined and digger_platforms != '' else 'android,ios' if groups['macos'] is defined and groups['macos'] | length > 0 else 'android' }}"
+
+- name: "Check for nagios deployment"
+  command: "oc get dc nagios --namepace={{ project_name }}"
+  register: nagios_dc
+  failed_when: "'NotFound' not in nagios_dc.stderr"
+
+- name: "Update dc with platforms"
+  command: "oc env dc/nagios PLATFORMS={{ digger_platforms }}"
```

Comment:
Its missing --namespace

Reply:
This needs to be passed the `--overwrite=true` flag also

Reply:
According to docs overwrite is true by default, but adding it explicitly won't hurt.

Reply:
This logic could be moved down after the `oc new-app` command in the same role and we do not need to check for dc at that point since we know it exists